CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -1,3 +1,12 @@
+# v1.9.2 - Changelog since v1.9.1
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Revert [#1090](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1090) because of labels regression ([#1174](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1174), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
I would phrase it more wrt the feature so the user knows what behavior is getting reverted.

Something like "Revert feature to add PV, PVC and namespace name as labels to the PD"

Reply:
Updated